### PR TITLE
Feature/gdb 7891 yasr download results in different format

### PR DESCRIPTION
**Pull request by boyan-tonchev**
## What?
Adds a "Download as" dropdown.

## Why?
Our clients want to download the result of an executed query in different formats.

## How?
Adds a "Download as" dropdown with different format options. When the user click on an option then "ontotext-yasgui-web-component" component will emit event wth payload the selected option, executed query and plugin name which is active at time when button is clicked.

### CODE REVIEW

**Comment by boyan-tonchev** [comment]
Jenkins retry

**Comment by boyan-tonchev** [comment]
Jenkins retry

**Comment by boyan-tonchev** [comment]
Jenkins retry

**Comment by boyan-tonchev** [comment]
Jenkins retry

**Comment by boyan-tonchev** [comment]
Jenkins retry

**Comment by boyan-tonchev** [comment]
Jenkins retry

**Comment by boyan-tonchev** [comment]
Jenkins retry

**Comment by boyan-tonchev** [comment]
Jenkins retry

**Comment by boyan-tonchev** [comment]
Jenkins retry

**Comment by boyan-tonchev** [comment]
Jenkins retry

File: ontotext-yasgui-web-component/src/components/dropdown/dropdown.tsx [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+  render() {
+    return (
+      <div class="ontotext-yasgui-dropdown">
+        <button class={`ontotext-yasgui-dropdown-button ${this.open ? 'icon-caret-up-after' : ' icon-caret-down-after'}`}
+                onClick={() => this.toggleComponent()}>
+          {this.translationService.translate(this.nameLabelKey)}
+        </button>
+
+        <ul class={`ontotext-yasgui-dropdown-menu ${this.open ? 'open' : 'closed'}`}>
```
Comment (svilenvelikov) [inline ontotext-yasgui-web-component/src/components/dropdown/dropdown.tsx:38]: This is not quite responsive. If there is not enough space on the right a horizontal scrollbar appears. It's probably not important to be handled right now, I just wanted to mention it.

File: ontotext-yasgui-web-component/src/components/dropdown/dropdown.tsx [hunk 2]
```diff
@@ -30,7 +30,7 @@ export class Dropdown {
   render() {
     return (
       <div class="ontotext-yasgui-dropdown">
-        <button class={`ontotext-yasgui-dropdown-button ${this.open ? 'icon-caret-up-after' : ' icon-caret-down-after'}`}
+        <button class={`ontotext-dropdown-button ${this.open ? 'icon-caret-up-after' : ' icon-caret-down-after'}`}
```
Comment (svilenvelikov) [inline ontotext-yasgui-web-component/src/components/dropdown/dropdown.tsx:33]: Can you please remove the `yasgui` from the dropdown below as well

Comment (boyan-tonchev) [reply ontotext-yasgui-web-component/src/components/dropdown/dropdown.tsx:33]: done

**Review by svilenvelikov — APPROVED** [review]

**Pull request merged**